components/PlantCard.js

```diff
@@ -50,17 +68,22 @@ export default function PlantCard({ plant }) {
       <ImageWrapper>
         <Image
           alt={`Image of ${plant.name}`}
-          layout="fill"
+          fill
           src={
             plant.imageUrl ||
             "https://images.unsplash.com/photo-1564502983799-becfbf817b4f?q=80&w=2070&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D"
           }
+          priority
```

Comment:
Why are you using `priority` here?
Do you think `priority` would lead to excessive resource loading on the initial page load at some point?